controllers/handlers/passt/passt.go

```diff
@@ -32,11 +34,10 @@ const (
 
 	passtCNIObjectName = "passt-binding-cni"
 
-	networkBindingNADName       = "primary-udn-kubevirt-binding"
-	networkBindingNADNamespace  = "default"
-	NetworkAttachmentDefinition = networkBindingNADNamespace + "/" + networkBindingNADName
-
+	networkBindingNADName        = "primary-udn-kubevirt-binding"
 	bindingComputeMemoryOverhead = "250Mi"
+
+	OpenShiftCNVNamespace = "openshift-cnv"
```

Comment:
As someone else said: "DON'T". Please remove this string from this PR, including from the PR title.